db/schema.rb

```diff
@@ -29,23 +29,8 @@
     t.string   "transaction_status"
     t.datetime "created_at",         null: false
     t.datetime "updated_at",         null: false
-  end
-
-  add_index "async_transactions", ["source_id"], name: "index_async_transactions_on_source_id", using: :btree
-  add_index "async_transactions", ["transaction_id", "source"], name: "index_async_transactions_on_transaction_id_and_source", unique: true, using: :btree
-  add_index "async_transactions", ["transaction_id"], name: "index_async_transactions_on_transaction_id", using: :btree
-  add_index "async_transactions", ["user_uuid"], name: "index_async_transactions_on_user_uuid", using: :btree
-
-  create_table "async_transactions", force: :cascade do |t|
-    t.string   "type"
-    t.string   "user_uuid"
-    t.string   "source_id"
-    t.string   "source"
-    t.string   "status"
-    t.string   "transaction_id"
-    t.string   "transaction_status"
-    t.datetime "created_at",         null: false
-    t.datetime "updated_at",         null: false
+    t.string   "metadata"
+    t.string   "metadata_vi"
```

Comment:
The diff overall for this file looks a bit odd, I think just because how the diff algorithm chose to show it. I'll give this and the migrations a smoke test first thing in the morning locally to verify that it all looks good on a different system.

Reply:
I think it reordered some things when I added the new columns. But I'm not sure why.